### ISSUE TITLE
Yarn detection fails when using `yarn policies set-version`

### ISSUE THREAD

**Issue by romandecker** [unicode-org/full-icu-npm#43]
Currently, `full-icu` detects whether or not the user is running yarn by matching the `npm_execpath` environment variable against `/yarn((.*cli)?\.js)?$/`.

If you're using [yarn policies](https://yarnpkg.com/lang/en/docs/cli/policies/) to ensure a consistent version of yarn, this variable will point to a checked-in version of `yarn`. The problem is, that the filename contains a version number by default, e.g.

```
❯ yarn run env | grep npm_execpath
  "npm_execpath": "/Users/user/projects/someproject/.yarn/releases/yarn-1.18.0.js",
```

This path does NOT match the regex, because of that version number, which leads to the install script thinking that it was called by `npm`, which in turn makes it fail with:

```
error /Users/user/projects/someproject/node_modules/full-icu: Command failed.
Exit code: 1
Command: node postinstall.js
Arguments:
Directory: /Users/user/projects/sps/someproject/node_modules/full-icu
Output:
npm install icu4c-data@64l (Node 12.13.1 and small-icu 64.2) -> icudt64l.dat
full-icu$ /Users/user/.nvm/versions/node/v12.13.1/bin/node /Users/user/projects/someproject/.yarn/releases/yarn-1.18.0.js install icu4c-data@64l
yarn install v1.18.0
info No lockfile found.
error `install` has been replaced with `add` to add new dependencies. Run "yarn add icu4c-data@64l" instead.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
/Users/user/projects/someproject/node_modules/full-icu/install-spawn.js:41
                throw(Error(cmdPath + ' ' + args.join(' ') + ' --> status ' + spawned.status));
```

**Comment by DeX3** [comment]
Hm, actually, it seems like the issue is already fixed in master, but not yet released. Would it be possible to release the fix?

**Comment by snowystinger** [comment]
bump

**Comment by srl295** [comment]
@romandecker v1.3.1 was published feb 24… closing this as I think that's what you referred to.

**Comment by srl295** [comment]
presumed fixed in #40
